packages/beacon-node/src/metrics/metrics/lodestar.ts

```diff
@@ -328,6 +328,27 @@ export function createLodestarMetrics(
       }),
     },
 
+    engineHttpProcessorQueue: {
+      length: register.gauge({
+        name: "lodestar_engine_http_processor_queue_length",
+        help: "Count of total engine http processor queue length",
+      }),
+      droppedJobs: register.gauge({
+        name: "lodestar_engine_http_processor_queue_dropped_jobs_total",
+        help: "Count of total engine http processor queue dropped jobs",
+      }),
+      jobTime: register.histogram({
+        name: "lodestar_engine_http_processor_queue_job_time_seconds",
+        help: "Time to process engine http processor queue job in seconds",
+        buckets: [0.1, 1, 10, 100],
+      }),
+      jobWaitTime: register.histogram({
+        name: "lodestar_engine_http_processor_queue_job_wait_time_seconds",
+        help: "Time from job added to the engine http processor queue to starting in seconds",
+        buckets: [0.1, 1, 10, 100],
```

Comment:
Bucket values must be realistic on expected values else not useful at all. See below as example of buckets values specific to a metric given expected values

https://github.com/ChainSafe/lodestar/blob/1cba8368ac433752d7f60df30fd0b551013226e3/packages/beacon-node/src/metrics/metrics/lodestar.ts#L369-L370

Reply:
hmmm right, most of the time will be taken up by new payload, and it can vary from 100s of ms to 2-4 seconds (if EL is really slow) and may be on average 300-400ms, so should be bucket for [0.1, 0.5, 2, 5] ?

Reply:
Then do for example
```
[0.05, 0.1, 0.2, 0.3, 0.5, 0.75, 1, 2, 5, 10, 25]
```